api/libproject/devstart.rb

```diff
@@ -1286,12 +1286,18 @@ def create_auth_domain(cmd_name, args)
   op.add_option(
     "--project [project]",
     ->(opts, v) { opts.project = v},
-    "Project to create the authorization domain"
+    "Workbench Project (environment) for creating the authorization domain"
   )
+  op.add_option(
+    "--user [user]",
+    ->(opts, v) { opts.user = v},
+    "A Workbench user you control with DEVELOPER Authority in the environment"
+  )
+  op.add_validator ->(opts) { raise ArgumentError unless opts.project and opts.user}
   op.parse.validate
 
   common = Common.new
-  common.run_inline %W{gcloud auth login}
+  common.run_inline %W{gcloud auth login #{op.opts.user}}
```

Comment:
Previously: opened a web browser requiring google login.  This was annoyingly disruptive when I had to run this more than once.

Reply:
It will still do this if it's the first time this user has logged into `gcloud` on your machine.  The next time, it won't.